framework/src/modules/random/schemas.ts

```diff
@@ -104,44 +104,66 @@ export type GetHashOnionUsageRequest = AddressRequest;
 export const getHashOnionUsageResponse = {
 	$id: 'lisk/random/getHashOnionUsageResponse',
 	type: 'object',
-	required: ['count', 'height', 'seed'],
+	required: ['usedHashOnions', 'seed'],
 	properties: {
-		count: {
-			type: 'integer',
-			format: 'uint32',
-		},
-		height: {
-			type: 'integer',
-			format: 'uint32',
+		usedHashOnions: {
+			type: 'array',
+			fieldNumber: 1,
```

Comment:
since this is JSON schema, instead of `fieldNumber` `dataType` it should use `type` and  `format` as it was